say/say.go

```diff
@@ -0,0 +1,84 @@
+package say
+
+import (
+	"fmt"
+	"strings"
+)
+
+var isDebug = false // override with --debug parameter
+
+func TurnOnDebugging() {
+	isDebug = true
+}
+
+func TurnOnDebuggingByArgs(args []string) {
+	// debug needs to be parsed at the beginning to have DEBUG enabled as quickly as possible
+	// otherwise, parsing others or other parameters don't have debug enabled
+	for i := 0; i < len(args); i++ {
+		if args[i] == "--debug" {
+			isDebug = true
+		}
+	}
+}
+
+func Error(text string) {
+	WithPrefix(text, "ERROR ")
+}
+
+func Warning(text string) {
+	WithPrefix(text, "⚠ ")
+}
+
+func Info(text string) {
+	WithPrefix(text, "> ")
+}
+
+func InfoIndented(text string) {
+	WithPrefix(text, "    ")
+}
+
+func Indented(text string) {
+	WithPrefix(text, "  ")
+}
+
+func Fix(instruction string, command string) {
+	WithPrefix(instruction, "👉 ")
+	EmptyLine()
+	Indented(command)
+	EmptyLine()
+}
+
+func Next(instruction string, command string) {
+	WithPrefix(instruction, "👉 ")
+	EmptyLine()
+	Indented(command)
+	EmptyLine()
+}
+
+func WithPrefix(s string, prefix string) {
+	lines := strings.Split(strings.TrimSpace(s), "\n")
+	for i := 0; i < len(lines); i++ {
+		PrintToConsole(prefix + strings.TrimSpace(lines[i]) + "\n")
+	}
+}
+
+func EmptyLine() {
+	PrintToConsole("\n")
+}
+
+func Say(s string) {
```

Comment:
I saw this is used, but should we really use it outside of this package? I am not sure...

Reply:
Its used a lot it seems. could rename it, but I think we need it.

Reply:
The name is fine but I was just thinking if this is something more general which should not be used. But for now it's fine to just leave it as it is.